### PR TITLE
Publications

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
Suggested text update and moving increasing visibility.

### CODE REVIEW

File: sites/docs/src/content/docs/guidelines/publication_credit.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Publication credit
+subtitle: Recognition for nf-core pipeline publications
+weight: 123
+---
+
+All pipeline publications and presentations, or those that describe pipelines for the first time, should give attribution to the nf-core community.
```
Comment (maxulysse) [inline sites/docs/src/content/docs/guidelines/publication_credit.md:7]:
```suggestion
All pipeline publications and presentations, or those that describe pipelines, should give attribution to the nf-core community.
```

File: sites/docs/src/content/docs/guidelines/publication_credit.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+**Optionally**, the nf-core community can be included as a consortium co-author. This option is preferred when a pipeline has made extensive use of existing nf-core pipeline components (e.g., modules and subworkflows) written by other community members who were not directly involved in the pipeline itself. For example, see hgtseq: [A Standard Pipeline to Study Horizontal Gene Transfer](https://doi.org/10.3390/ijms232314512).
+
+If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation.
+
+If your pipeline publication or presentation is an extension of a community pipeline, please consider adding the main developers as coauthors.
```
Comment (maxulysse) [inline sites/docs/src/content/docs/guidelines/publication_credit.md:21]:
```suggestion
If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.
```

**Comment by ewels** [comment]
Text looks good, and I get why moving it gives it more visibility. But this is still a pipeline guideline for me. It's something that should be adhered to for all pipelines, like the other guidelines. I'm not totally convinced that moving it up a level feels right..?

**Comment by christopher-hakkaart** [comment]
I guess my issue is that it's buried pretty deep in the navigation ( Guidelines > Pipelines >  Recommendations > Pipeline credit ). If you didn't know it's there it's hard to find.

**Comment by christopher-hakkaart** [comment]
I've moved the page back to pipeline recommendations.

**Review by jfy133 — APPROVED** [review]

**Pull request merged**